matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/mapper/RoomMemberSummaryMapper.kt

```diff
@@ -18,12 +18,14 @@ package org.matrix.android.sdk.internal.database.mapper
 
 import org.matrix.android.sdk.api.session.room.model.RoomMemberSummary
 import org.matrix.android.sdk.internal.database.model.RoomMemberSummaryEntity
+import org.matrix.android.sdk.internal.database.model.presence.toUserPresence
 
 internal object RoomMemberSummaryMapper {
 
     fun map(roomMemberSummaryEntity: RoomMemberSummaryEntity): RoomMemberSummary {
         return RoomMemberSummary(
                 userId = roomMemberSummaryEntity.userId,
+                userPresence = roomMemberSummaryEntity.userPresence?.toUserPresence(),
```

Comment:
`userPresence?.toUserPresence()` is a bit strange to read. Maybe rename `roomMemberSummaryEntity.userPresence` to `roomMemberSummaryEntity.userPresenceEntity`